frontend/src/pages/Contracts/Contract/Amendments/editFields.tsx

```diff
@@ -0,0 +1,30 @@
+import { IEditFields } from "types";
```

Comment:
we discussed this, makes sense to put both editFields and readFields into a file called fields